xbmc/interfaces/json-rpc/VideoLibrary.cpp

```diff
@@ -1126,8 +1126,11 @@ void CVideoLibrary::UpdateVideoTag(const CVariant &parameterObject, CVideoInfoTa
   if (ParameterNotNull(parameterObject, "rating"))
   {
     details.SetRating(parameterObject["rating"].asFloat());
-    if (ParameterNotNull(parameterObject, "votes"))
-      details.SetVotes(StringUtils::ReturnDigits(parameterObject["votes"].asString()));
+    updatedDetails.insert("ratings");
+  }
+  if (ParameterNotNull(parameterObject, "votes"))
+  {
+    details.SetVotes(StringUtils::ReturnDigits(parameterObject["votes"].asString()));
     updatedDetails.insert("ratings");
```

Comment:
Can you add a small comment here why this is needed? Might be confusing and not everybody might want to look what's happening here.

Rest seems fine.